server/migrations/20190603135309-create-user.js

```diff
@@ -0,0 +1,54 @@
+const userMigration = {
```

Comment:
This is already in the migrations directory. There is no need to call it userMigrations

Reply:
Find a way to rename the migration file appropriately